pkg/systemd/services/services.jsx

```diff
@@ -405,9 +418,8 @@ class ServicesPage extends React.Component {
                                         .then(result => {
                                             // First add the path in the seens paths to maintain the loading state
                                             result.forEach(path => this.seenPaths.add(path));
-                                            const allPaths = paths.concat(result);
 
-                                            return Promise.all(allPaths.map(path => this.getUnitByPath(path)));
+                                            return Promise.all(result.map(path => this.getUnitByPath(path)));
```

Comment:
Why are there calls to LoadUnit and getUnitByPath at all in listUnits? Wasn't the goal to get rid of all of them?

Reply:
For the units that are not listed by ListUnits but only by ListUnitFiles, we still need to get some information for these - like Description to show in the main page.